### PR TITLE
[INTEGRATION] [dbt] parse dbt command line arguments that override defaults

### PR DESCRIPTION
**Pull request by mobuchowski**
dbt run has few command line arguments that change location of artifacts, like `--project-dir` or ones that override defaults, like `--profile`. This PR adds support for few of them that can affect lineage event generation. 

Solves issue 2) in https://github.com/OpenLineage/OpenLineage/issues/193
Signed-off-by: Maciej Obuchowski <maciej.obuchowski@getindata.com>

### CODE REVIEW

**Comment by julienledem** [comment]
This looks good. I think the other aspect of 2) in  #193 that is not covered here is that jobs should have a different name depending on which schema they are run against.
When you pick a different `target` in your `profile` (as defined [in the dbt doc here](https://docs.getdbt.com/dbt-cli/configure-your-profile#understanding-profiles) ) the whole project will run on a different `dbname` and `schema`. Which means that the jobs should have a different name depending on which db/schema they are run against (as they have a different lineage).

**Comment by mobuchowski** [comment]
@julienledem addressed that in the following commit.

Job names are now `{database}.{schema}.{model_name}` without `model.`

**Review by julienledem — APPROVED** [review]
awesome!

File: integration/common/openlineage/common/utils.py [hunk 1]
```diff
@@ -50,3 +50,17 @@ def get_from_multiple_chains(source: Dict[str, Any], chains: List[List[str]]) ->
         if result:
             return result
     return None
+
+
+def parse_single_arg(args, keys: List[str]) -> Optional[str]:
```
Comment (wslulciuc) [inline integration/common/openlineage/common/utils.py:55]: Why not use [`argparse`](https://docs.python.org/3/library/argparse.html)?

**Review by wslulciuc — APPROVED** [review]
Left a minor comment on how we are parsing the cmd line args, otherwise 👍

File: integration/common/openlineage/common/utils.py (hunk 1 above)
Comment (mobuchowski) [reply integration/common/openlineage/common/utils.py:55]: I thought about it, but it's far too complex for this simple use case and requires disabling a lot of options, like help, exit on failure or usage. This function does exactly what we need, which is transparent searching for keys in passed list.

**Comment by julienledem** [comment]
@mobuchowski We should default the namespace to the same namespace as the output dataset (as you have in the 0.1.0 release).

**Pull request merged**